am-lib/src/main/java/uk/gov/hmcts/reform/amlib/repositories/AccessManagementRepository.java

```diff
@@ -41,4 +43,8 @@
     @SqlQuery("select * from access_management where accessor_id=? and resource_id=? LIMIT 1")
     @RegisterConstructorMapper(ExplicitAccessRecord.class)
     ExplicitAccessRecord getExplicitAccess(String accessorId, String resourceId);
+
+    @SqlQuery("select * from default_permissions_for_roles where service_name = :serviceName and resource_type = :resourceType and resource_name = :resourceName and role_name = :roleName")
+    @RegisterConstructorMapper(RoleBasedAccessRecord.class)
+    List<RoleBasedAccessRecord> getRolePermissions(@BindBean RoleBasedAccessGrant roleBasedAccessGrant);
```

Comment:
Why that method accepts some sort of grant? Why not just 4 string args?

Reply:
PMD does not allow it, it says too many strings and so asks for it to be extracted.

Reply:
👍